feat(changelog): containers-added-serverless-network-metrics 2024-11-05 (#3945)
scaleway/docs-content: changelog/november2024/2024-11-05-containers-added-serverless-network-metrics.mdx

---

> title: Serverless network metrics

Reviewer: ```suggestion
title: Serverless network metrics are evolving
```

status: added
author:
    fullname: 'Join the #serverless-containers channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-11-05
category: serverless
product: containers
---

> In the Serverless Containers metrics panel we have added access to network information on received and transmitted data.

Reviewer 2: ```suggestion
In the Serverless Containers metrics panel, we added access to network information on received and transmitted data. Refer to the [dedicated documentation](/serverless/containers/how-to/monitor-container/) for more information on how to monitor a container.
```

> Metrics are evolving, in Serverless Containers metrics panel we added access to network informations on received data and transmitted data.

Reviewer: ```suggestion
In the Serverless Containers metrics panel we have added access to network information on received and transmitted data.
```
